source/amazon/services/prerequisites/credentials.rst

```diff
@@ -181,20 +180,3 @@ If you're using a single AWS account for all your buckets this could be the most
 
 * ``AWS_ACCESS_KEY_ID``
 * ``AWS_SECRET_ACCESS_KEY``
-
-Insert the credentials into the configuration
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. deprecated:: 4.4.0
-
-Another available option to set up credentials is writing them right into the Wazuh configuration file (``/var/ossec/etc/agent.conf``), inside of the ``<bucket>`` block on the module configuration.
-
-This is an example configuration:
-
-.. code-block:: xml
-
-  <bucket type="cloudtrail">
-    <name>my-bucket</name>
-    <access_key>insert_access_key</access_key>
-    <secret_key>insert_secret_key</secret_key>
-  </bucket>
```

Comment:
According to what I said above, this should remain in order to be merged in a `path` version, so only examples should be removed.